### PR TITLE
Split into two images (vagrant), save almost 1GB for our most used image and updates

### PR DESCRIPTION
**Pull request by majst01**
- Updates to ansible 2.9.15
- Updates to Vagrant 2.2.14

### CODE REVIEW

File: metal-deployment/base/Dockerfile [hunk 1]
```diff
@@ -1,49 +1,39 @@
...
- && pip install --upgrade pip \
- && pip install ansible==${VERSION_ANSIBLE} Jinja2==2.11.1 netaddr==0.7.19 humanfriendly==8.1 openshift==0.11.2 paramiko==2.7.1 \
- && curl -fo vagrant.deb $VAGRANT_PACKAGE_URL \
- && dpkg -i vagrant.deb \
- && rm -f vagrant.deb \
- && vagrant plugin install vagrant-libvirt \
- && curl -Lo stern https://github.com/wercker/stern/releases/download/${VERSION_STERN}/stern_linux_amd64 \
- && chmod +x stern \
- && mv stern /usr/local/bin/ \
+ && pip3 install ansible==${VERSION_ANSIBLE} Jinja2==2.11.1 netaddr==0.7.19 humanfriendly==8.1 openshift==0.11.2 paramiko==2.7.1 \
```
Comment (Gerrit91) [inline metal-deployment/base/Dockerfile:31]: We often execute `pip` in CI jobs to install something specific for testing. `pip` has to be available.

**Review by Gerrit91 — CHANGES_REQUESTED** [review]
Can we append `vagrant` to the image tags and create an individual image name? It's a bit easier to introduce this when you just have to append something at the end of the string and not "in-between". Also it's closer to how big images do it on Docker Hub (e.g. Postgres has postgres:12-alpine).

File: metal-deployment/base/Dockerfile (hunk 1 above)
Comment (majst01) [reply metal-deployment/base/Dockerfile:32]: Good ideas, done

**Comment by Gerrit91** [comment]
Works for mini-lab now with the `-vagrant` image.

**Review by Gerrit91 — APPROVED** [review]

**Pull request merged**